app/assets/javascripts/initializers/initializeVideoPlayback.js

```diff
@@ -163,14 +170,16 @@ function initializeVideoPlayback() {
     } else {
       // jwplayer is initialized and no further interaction is needed
       initWebPlayer(seconds, metadata);
+      var properties = { article: metadata.id, deviceType: deviceType };
+      ahoy.track('Article video play', properties);
```

Comment:
Hm, it's a little weird to me that we're calling `ahoy.track` within the `initializePlayer`. That means that we'd be creating a new `Ahoy::Event` every single time the player was initialized, which I think isn't quite what we want to do when we track events? I think it makes sense to just remove this given where it is being called, but maybe I am missing some context here, so please feel free to fill me in if I'm missing something obvious :)

Reply:
That's a great question as it's not obvious from the `init/initialized` naming used. The idea behind the event was to know when users "watch a video" for all platforms (web/Android/iOS). 

For both mobile platforms the event is triggered on a click/tap, so the new name for the event suggested there (`Clicked Video Player`) makes a lot of sense. On the web browser the video player is initialized with autoplay, so there will be no click event to track when the video starts to play.

__Follow-up question:__ @vaidehijoshi would it be better to use an event name like `Video Player Started`/`Started Streaming Video`/etc? Or maybe just remove this event from the web entirely? I wouldn't want to get rid of the web tracking so we can compare across all platforms, but if it's not the intended use for Ahoy I can remove it.

Naming things is hard 😆

Reply:
@fdoxyz Agree re: naming things being hard 🙈 

This additional context is super helpful, thanks for providing it! 😃  We could definitely name the event differently (I like `Video Player Streaming Started` or something like you suggested!), if that is something we still want to track on the web side and if we can't actually "track" a click in that case (it sounds like we can't since the web video player plays automatically).

This makes me wonder if we'd ever want to know when someone on the web platform pauses/stops a video? I'm thinking about this because, if web videos are just automatically streamed without the user triggering a click event to start them, do we want to know how many of those "started" videos get stopped or paused immediately?

Reply:
This is great feedback thank you @vaidehijoshi! 

Looking back at the Podcast Player [here](https://github.com/forem/forem/blob/master/app/assets/javascripts/initializers/initializePodcastPlayback.js#L371) __we do track play/pause events__ (next in line for a refactor to remove GA in favor of Ahoy). I took another deep look at `jwplayer` and found an event that we can hook onto in order to get these updates (with a small workaround).

Just sent in a new commit with the event name `Video Player Streaming` and the properties payload now look like this:

```
{"article":29,"deviceType":"Android","action":"pause"}
```

I was also considering `Video Player Interaction` for the name, but me know what you think 🙂